CHANGELOG.md

```diff
@@ -1,111 +0,0 @@
-# Changelog
```

Comment:
Why did you delete the CHANGELOG.md file? Shouldn't we continue to have changes accumulate into this log file?

Reply:
I removed CHANGELOG.md and release.md as the release process automatically creates a release description and change log using Git commit history. I also observed that when you created the 1.8.2, that these files were not updated.. Just trying to take friction out of the release process. If you feel we need more "history" than what git provides, let me know. My preference would be to leave history to the change control system.